crates/bevy_window/src/lib.rs

```diff
@@ -130,14 +131,16 @@ impl Plugin for WindowPlugin {
             .add_event::<AppLifecycle>();
 
         if let Some(primary_window) = &self.primary_window {
-            let mut entity_commands = app.world_mut().spawn(primary_window.clone());
-            entity_commands.insert((
-                PrimaryWindow,
-                RawHandleWrapperHolder(Arc::new(Mutex::new(None))),
-            ));
-            if let Some(primary_cursor_options) = &self.primary_cursor_options {
-                entity_commands.insert(primary_cursor_options.clone());
-            }
+            let primary_window = primary_window.clone();
+            let primary_cursor_options = self.primary_cursor_options.clone().unwrap_or_default();
```

Comment:
This is the same behavior as before because it is a required component of `Window`